code/modules/food_and_drinks/food/foods/meat.dm

```diff
@@ -47,6 +47,12 @@
 	desc = "Much meatier than monkey meat."
 	list_reagents = list("nutriment" = 5, "vitamin" = 1)
 
+/obj/item/food/snacks/meat/kangaroo
+	name = "kangaroo meat"
+	desc = "Extremely muscular and tender meat."
+	list_reagents = list("protein" = 4, "iron" = 5, "vitamin" = 1)
+	tastes = list("a punch in the face" = 1, "fowl" = 3)
```

Comment:
I still don't think it should taste like chicken, according to google the only thing close to it tasting like chicken is that they're both white meats? Even that I'm not sure of

Reply:
According to google it tastes like game, which I can confirm, is very unlike chicken

Reply:
i dont know how it taste so i open to improve. This is not important.